develop/dev-guide-third-party-support.md

```diff
@@ -26,48 +26,51 @@ If you encounter problems when connecting to TiDB using the tools listed in this
-| Python                | [Django](https://pypi.org/project/Django/)                                                  | v4.0.5      | Compatible | N/A                                                    | N/A                                                                            |
-| Python                | [peewee](https://github.com/coleifer/peewee/)                                                                                                                                 | v3.14.10    | Compatible | N/A                                                    | N/A                                                                            |
-| Python                | [PonyORM](https://ponyorm.org/)                                                                                                                                               | v0.7.16     | Compatible | N/A                                                    | N/A                                                                            |
-| Python                | [SQLAlchemy](https://www.sqlalchemy.org/)                                                                                                                                     | v1.4.37     | Compatible | N/A                                                    | N/A                                                                            |
+| JavaScript/TypeScript | [sequelize](https://www.npmjs.com/package/sequelize) | v6.20.1 | Compatible | N/A | N/A |
+| JavaScript/TypeScript | [Knex.js](https://knexjs.org/) | v1.0.7 | Compatible | N/A | N/A |
+| JavaScript/TypeScript | [Prisma Client](https://www.prisma.io/) | 3.15.1 | Compatible | N/A | N/A |
+| JavaScript/TypeScript | [TypeORM](https://www.npmjs.com/package/typeorm) | v0.3.6 | Compatible | N/A | N/A |
+| PHP | [laravel](https://laravel.com/) | v9.1.10 | Compatible | [laravel-tidb](https://github.com/colopl/laravel-tidb) | N/A |
+| Python | [Django](https://pypi.org/project/Django/) | v4.0.5 | Compatible | [django-tidb](https://github.com/pingcap/django-tidb) | N/A |
+| Python | [peewee](https://github.com/coleifer/peewee/) | v3.14.10 | Compatible | N/A | N/A |
+| Python | [PonyORM](https://ponyorm.org/) | v0.7.16 | Compatible | N/A | N/A |
+| Python | [SQLAlchemy](https://www.sqlalchemy.org/) | v1.4.37 | Compatible | N/A | N/A |
 
 ## GUI
 
-| GUI                                           | Latest tested version  | Support level | Tutorial  |
-|-----------------------------------------------|---------|------|-----|
-| [DBeaver](https://dbeaver.io/)                | 22.1.0  | Compatible | N/A |
+| GUI | Latest tested version | Support level | Tutorial |
+| - | - | - | - |
+| [DBeaver](https://dbeaver.io/) | 22.1.0 | Compatible | N/A |
 | [Navicat for MySQL](https://www.navicat.com/) | 16.0.14 | Compatible | N/A |
```

Comment:
```suggestion
| [Navicat for MySQL](https://www.navicat.com/) | 16.0.14 | Compatible | N/A |
| [MySQL Workbench](https://www.mysql.com/products/workbench/) | 8.0 | Compatible | N/A |
```

Workbench is mentioned already on https://docs.pingcap.com/tidb/stable/mysql-compatibility